File: Ages.py
fathers_ages = int(input("Enter current father's age (1-80):"))
son_ages = int(input("Enter current age of son (1-80):"))

years = abs (fathers_age -2 * son_age)


if years == 0:
	print("The father is currently twice as old as his son.")
elif father_age > 2 * son_age:
	print(f" The father was twice as old as his son years ago.")

else:	
	print(f"The father will be twice as old as his son in years.")

File: Ages.java
//prompt a user to take two inputs(father's age,son's age) and
 //calculate how many years ago or in future father was twice as son.





import java.util.Scanner;

public class Ages {
  public static void main(String[] args) {

	Scanner input = new Scanner(System.in);

	System.out.print("Enter current father's age (1-80):");
	int father = input.nextInt();

	System.out.print("Enter current age of son (1-80):");
	int son = input.nextInt();
	
         int years = Math.abs(father -2 *son);
	

	if (years == 0) {
		 System.out.println("The father is currently twice as old as his son.");

	} else if (father > 2 * son) {
		 System.out.println("THe father was twice old as his son in " + years + " years.");
	}


    }


}